src/components/handbook-navigation/content.js

```diff
@@ -4,12 +4,13 @@ export const content = [
   { id: 2, link: "/community/handbook/community-roles", text: "Roles" },
   { id: 3, link: "/community/handbook/contribution", text: "Contribution" },
   { id: 4, link: "/community/handbook/recogition", text: "Recognition" },
-  { id: 5, link: "/community/handbook/repository-overview", text: "Repository Overview" },
-  { id: 6, link: "/community/handbook/projects", text: "Projects" },
-  { id: 7, link: "/community/handbook/mentorship-programs", text: "Mentorship Programs" },
-  { id: 8, link: "/community/handbook/writing-program", text: "Writing Program" },
-  { id: 9, link: "/community/handbook/learn-layer5", text: "Learning" },
-  { id: 10, link: "/community/handbook/connect-with-us", text: "Connect with us" },
-  { id: 11, link: "/community/handbook/code-of-conduct", text: "Code of Conduct" },
+  { id: 4, link: "/community/handbook/repository-overview", text: "Repository Overview" },
+  { id: 5, link: "/community/handbook/projects", text: "Projects" },
+  { id: 6, link: "/community/handbook/mentorship-programs", text: "Mentorship Programs" },
+  { id: 7, link: "/community/handbook/writing-program", text: "Writing Program" },
+  { id: 8, link: "/community/handbook/learn-layer5", text: "Learning" },
+  { id: 9, link: "/community/handbook/connect-with-us", text: "Connect with us" },
+  { id: 10, link: "/community/handbook/code-of-conduct", text: "Code of Conduct" },
+  { id: 11, link: "/community/handbook/security-vulnerabilities", text: "Security Vulnerabiities" },
```

Comment:
The id values are incorrect. It should start from 5

Reply:
// @gurpreet-legend

Reply:
> While you are there, please fix the unwanted `padding-top` to the `h2` elements in the handbook.

@Nikhil-Ladha , I have fixed the indexing, it might have been because of a merge.
Also can you tell me more about which padding you are referring to here, a screenshot might help :)

Reply:
Ahh...I should have shared a ss earlier itself, my bad.
Here you go, the extra whitespace above the heading "About Layer5" is what I am referring to.
<img width="1483" alt="Screenshot 2023-02-22 at 11 57 24 PM" src="https://user-images.githubusercontent.com/29438875/220724639-50f85444-d40e-4991-b373-fa162acc0f11.png">

Reply:
@Nikhil-Ladha , all the changes are done, it's ready to merge :)